packages/dotlottie-js/src/common/dotlottie-common.ts

```diff
@@ -245,18 +246,33 @@ export class DotLottieCommon {
     });
   }
 
-  private _renameImageAssets(): void {
-    const images: Map<string, LottieImageCommon[]> = new Map();
+  /**
+   * Generates a map of duplicate image ids and their count.
+   * @returns Map of duplicate image ids and their count.
+   */
+  private _getMapOfDuplicateImageIds(): Map<string, number> {
+    let count = 0;
```

Comment:
Why do we need this global `count` variable ? isn't the map already has the occurrence of each image by its ID ?